content/docs/run-api-server/Ingestion-filtering.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Ingestion Filtering
+order: 46
+---
+
+The Ingestion Filtering feature is now released for public beta testing available from Horizon [version 2.18.0](https://github.com/stellar/go/releases/tag/horizon-v2.18.0) and up
+  
+## Overview
+
+Ingestion Filtering enables Horizon operators to drastically reduce storage footprint of their Horizon DB by whitelisting Assets and/or Accounts that are relevant to their operations. This feature is ideally suited for private Horizon operators who do not need full history for all assets and accounts on the Stellar network. 
+
+### Why is it useful: 
+
+Previoulsy, the only way to limit data storage is by limiting the amount of history Horizon ingests, either by configuring the starting ledger to be later than genesis block or via rolling retention (ie: last 30 days). This feature allows users to store the full history of assets and accounts (and related entities) that they care about. 
+
+For further context, running a full history Horizon instance currently takes ~ 15TB of disk space (as of June 2022) with storage growing at a rate of ~ 1TB / month. As a benchmark, filtering by even 100 of the most active accounts and assets reduces storage by over 90%. For the majority of users who care about an even more limited set of assets and accounts, storage savings should be well over 99%. Other benefits are reducing operating costs for maintaining storage, improved DB health metrics and query performance. 
+
+### How does it work:
+
+This feature provides an ability to select which ledger transactions are accepted at ingestion time to be stored in Horizon’s historical database. Filter whitelists are maintained via an admin REST API (and persisted in the DB). The ingestion process checks the list and persists transactions related to Accounts and Assets that are whitelisted. Note that the feature does not filter the current state of the ledger and related DB tables, only history tables. 
+ 
+
+Whitelisting can include the following supported entities:
+
+- Account id
+- Asset id (canonical)
+
+Given that all transactions related to the white listed entities are included, all historical time series data related to those transactions are saved in horizon's history db as well. For example, whitelisting an Asset will also persist all Accounts that interact with that Asset and vice versa, if an Account is whitelisted, all assets that are held by that Account will also be included. 
+
+## Configuration:
+The filters and their configuration are optional features and must be enabled with horizon command line or environmental parameters: 
+
+<CodeExample>
+  
+```
+admin-port=<your_choice> 
+```
+</CodeExample>
+
```

Comment:
iirc you need a full blank line before and after the opening and closing custom tags like `<CodeExample>` otherwise the build will break